src/corehost/common/pal.h

```diff
@@ -50,16 +50,26 @@
 
 #endif
 
-
+// When running on a platform that is not supported in RID fallback graph (because it was unknown
+// at the time the SharedFX in question was built), we need to use a reasonable fallback RID to allow
+// consuming the native assets.
+//
+// For Windows and OSX, we will maintain the last highest RID-Platform we are known to support for them as the 
+// degree of compat across their respective releases is usually high.
+//
+// We cannot maintain the same (compat) invariant for linux and thus, we will fallback to using lowest RID-Plaform.
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define FALLBACK_HOST_RID _X("win10")
```

Comment:
Sorry I missed that big conversation as it got folded away in the code changes.

While I agree we would miss some assets it was my understanding that we would change all our RID packages to be based on "win" and "osx" and we would push others to follow a similar pattern and when that happens this wouldn't be a problem any longer. To me it is the exact same issue we are trying to solve with moving to "linux".

I'm curious to get @ericstj and @Petermarcu opinion here as I'm not hung up on it but it would be nice to be consistent across platforms.